Dangerfile

```diff
@@ -29,10 +29,9 @@ Dir[lint_dir].each do |file_name|
   android_lint.lint(inline_mode: true)
 end
   
-# TODO: Update to support test coverage report from Jacoco for Coroutine Template
-# Show Danger test coverage report from Jacoco for RxJavaTemplate
-jacoco_dir = "RxJavaTemplate/**/build/reports/jacoco/jacocoTestReport/jacocoTestReport.xml"
-markdown "## RxJavaTemplate Jacoco report:"
```

Comment:
Hmmmmm, what's the reason we removed it here?

I was planning to remove it in [this pull request](https://github.com/nimblehq/android-templates/pull/175)

Reply:
Actually I'm fine to keep it, my guess is that if we want to keep both it will require more changes, right? 😁

Reply:
Because I would like to apply Danger report for this template but it seems we can not report for both in the same time. Anw, you or me will need to rebase, first come first serve 👋

Reply:
@luongvo Sorry for the conflicts bro 😁